statsmodels/tsa/statespace/tests/test_sarimax.py

```diff
@@ -1170,16 +1167,23 @@ def setup_class(cls, *args, **kwargs):
         # of states if enforce_stationarity = False
         cls.model.ssm.loglikelihood_burn = 0
 
-    def test_loglike(self):
```

Comment:
This test seems to pass fine using the version in the parent class (pasted below).  It seem that the reduced tolerance wasn't necessary, and so this is just a little clean up.  The `init_keys_replication` needed some fixed to pass.  Not sure if these are strictly kosher or not -- "fixing the test" might be proving that it doesn't work as expected.

```
    def test_loglike(self):
        self.result = self.model.filter(self.true_params)

        assert_allclose(
            self.result.llf,
            self.true_loglike,
            atol=0.7 * 10**(-self.decimal)
        )
```